### PR TITLE
[FIX] update query to find active users on `Team.members`

### PR DESCRIPTION
**Pull request by LucianoPierdona**
## Proposed changes (including videos or screenshots)

This PR fixes an issue where previously if no query was passed through, the query to find active users was looking for all users instead of only team members

### CODE REVIEW

File: apps/meteor/server/services/team/service.ts [hunk 1]
```diff
@@ -633,14 +633,19 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
...
 		}
 
-		const users = await Users.findActive({ ...query }).toArray();
-		const userIds = users.map((m) => m._id);
-		const cursor = TeamMember.findMembersInfoByTeamId(teamId, count, offset, {
-			userId: { $in: userIds },
-		});
+		const cursor = this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset);
+		const records = await cursor.toArray();
+		const teamUserIds = records.map((m) => m.userId);
```
Comment (matheusbsilva137) [inline apps/meteor/server/services/team/service.ts:638]:
```suggestion
		const teamMembers = await this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset).toArray();
		const teamUserIds = teamMembers.map((m) => m.userId);
```
Maybe we could change the const names here to improve legibility

File: apps/meteor/server/services/team/service.ts [hunk 2]
```diff
@@ -633,14 +633,19 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
...
+
+		query._id = {
+			$in: teamUserIds,
+		};
```
Comment (matheusbsilva137) [inline apps/meteor/server/services/team/service.ts:642]: I think we should check if `query._id` is undefined here before assigning this new value so as to prioritize the user's query (if there is any)

**Review by matheusbsilva137 — CHANGES_REQUESTED** [review]

File: apps/meteor/server/services/team/service.ts (hunk 2 above)
Comment (LucianoPierdona) [reply apps/meteor/server/services/team/service.ts]: Updated!

**Comment by LucianoPierdona** [comment]
Close PR due to fixes have been applied on other PR.

**Pull request closed without merging**